core/go.mod

```diff
@@ -4,27 +4,66 @@ go 1.21
 
 require (
 	cosmossdk.io/log v1.3.1
+	github.com/cometbft/cometbft v0.38.7
```

Comment:
big no no

Reply:
So we shouldn't import comet into core?

Reply:
Yes, no comet, no sdk and as little as dep as possible in core.